go/obscuronode/header.go

```diff
@@ -0,0 +1,63 @@
+package obscuronode
```

Comment:
I imagined this to be a super dumb storage, similar to the "db" we have in the mock l1 and in the enclave.
Just an indexed storage from:
hash -> rollup header
hash -> block
and a pointer to the current heads

That would give us flexibility later to replace it with a real database.

Reply:
Refactored 👍